daemon/cmd/daemon_main.go

```diff
@@ -580,6 +580,12 @@ func init() {
 	flags.String(option.MaglevHashSeed, maglev.DefaultHashSeed, "Maglev cluster-wide hash seed (base64 encoded)")
 	option.BindEnv(option.MaglevHashSeed)
 
+	flags.Uint(option.MaglevMaxBackends, maglev.DefaultMaxBackends,
```

Comment:
One more nit, could we rename `MaglevMaxBackends` to something that would better reflect that it's used to control the permutation slice pre-allocation and does not set a limit for backends count?

Reply:
Sorry for being late to look closer here. Can we derive this heuristic from elsewhere without an additional agent flag? I think such flag is just confusing to users and more overhead to understand what needs to be configured and what not. For example, we already have M table size which should be good enough to make a heuristic / assumption about the potential scale of backends for the worst case. Lets please not add more agent knobs iff we can potentially avoid it.